modules/gitops-release-notes-1-5-0.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-5-0_{context}"]
+= Release notes for {gitops-title} 1.5.0
+
+[role="_abstract"]
+{gitops-title} 1.5.0 is now available on {product-title} 4.8, 4.9 and 4.10.
+
+[id="new-features-1-5-0_{context}"]
+== New features
+
+The current release adds the following improvements.
+
+* This enhancement upgrades Argo CD to version *2.3.3*. link:https://issues.redhat.com/browse/GITOPS-1708[GITOPS-1708]
+
+* This enhancement upgrades Application Set to version *0.4.1*. link:https://issues.redhat.com/browse/GITOPS-1850[GITOPS-1850]
+
+* This enhancement upgrades Dex to version *2.30.3*. link:https://issues.redhat.com/browse/GITOPS-1850[GITOPS-1850]
+
+* This enhancement upgrades Helm to version *3.8.0*. link:https://issues.redhat.com/browse/GITOPS-1709[GITOPS-1709]
+
+* This enhancement upgrades Kustomize to version *4.4.1*. link:https://issues.redhat.com/browse/GITOPS-1710[GITOPS-1710]
+
+* With this update, a new channel as *latest* has been added where all the latest version releases will be updated. For GitOps v1.5.0, the operator will be pushed to gitops-1.5, *latest* channel, and the existing *stable* channel. link:https://issues.redhat.com/browse/GITOPS-1791[GITOPS-1791]
+
```

Comment:
We should mention that we won't be pushing to the "stable" channel starting in v1.6. (We want to give users some advance notice.)